From 2db350fa3a9d599b72bf21503059b01340c778ee Mon Sep 17 00:00:00 2001
From: raveit65 <mate@raveit.de>
Date: Sat, 6 Apr 2019 13:36:00 +0200
Subject: [PATCH 4/6] Menta: improve metacity-theme-3

- increase border-width for frame-geometry "border"
- increase title_vertical_pad for frame-geometry "modal"
- draw buttons on dialog windows
- fix glitch for focused modal_dialog windows
---
 .../BlueMenta/metacity-1/metacity-theme-3.xml    | 16 ++++++++--------
 .../Menta/metacity-1/metacity-theme-3.xml        | 16 ++++++++--------
 2 files changed, 16 insertions(+), 16 deletions(-)

diff --git a/desktop-themes/BlueMenta/metacity-1/metacity-theme-3.xml b/desktop-themes/BlueMenta/metacity-1/metacity-theme-3.xml
index 470bed0d..063ebb4b 100644
--- a/desktop-themes/BlueMenta/metacity-1/metacity-theme-3.xml
+++ b/desktop-themes/BlueMenta/metacity-1/metacity-theme-3.xml
@@ -77,9 +77,9 @@
 </frame_geometry>
 
 <frame_geometry name="border" has_title="false" rounded_top_left="false" rounded_top_right="false" parent="normal" >
-	<distance name="left_width" value="1" />
-	<distance name="right_width" value="1" />
-	<distance name="bottom_height" value="1" />
+	<distance name="left_width" value="6" />
+	<distance name="right_width" value="6" />
+	<distance name="bottom_height" value="6" />
 	<border name="title_border" left="10" right="10" top="0" bottom="0" />
 	<border name="button_border" left="0" right="0" top="0" bottom="0"/>
 	<distance name="title_vertical_pad" value="1" />
@@ -94,8 +94,8 @@
 	<border name="title_border" left="0" right="0" top="0" bottom="0" />
 </frame_geometry>
 
-<frame_geometry name="modal" title_scale="small" hide_buttons="true" rounded_top_left="false" rounded_top_right="false" parent="small">
-	<distance name="title_vertical_pad" value="5"/>
+<frame_geometry name="modal" title_scale="medium" hide_buttons="false" rounded_top_left="false" rounded_top_right="false" parent="small">
+	<distance name="title_vertical_pad" value="10"/>
 </frame_geometry>
 
 <frame_geometry name="attached" title_scale="medium" hide_buttons="true" rounded_top_left="4" rounded_top_right="4" rounded_bottom_left="4" rounded_bottom_right="4" parent="normal">
@@ -645,7 +645,7 @@
 	<button function="unstick" state="pressed"><draw_ops></draw_ops></button>
 </frame_style>
 
-<frame_style name="dialog_focused" geometry="nobuttons">
+<frame_style name="dialog_focused" geometry="normal">
 	<piece position="entire_background" draw_ops="entire_background_focused" />
 	<piece position="titlebar" draw_ops="rounded_titlebar_focused" />
 	<piece position="title" draw_ops="title_focused" />
@@ -677,7 +677,7 @@
 	<button function="unstick" state="pressed"><draw_ops></draw_ops></button>
 </frame_style>
 
-<frame_style name="dialog_unfocused" geometry="nobuttons">
+<frame_style name="dialog_unfocused" geometry="normal_unfocused">
 	<piece position="entire_background" draw_ops="entire_background_unfocused" />
 	<piece position="titlebar" draw_ops="rounded_titlebar_unfocused" />
 	<piece position="title" draw_ops="title_unfocused" />
@@ -710,7 +710,7 @@
 
 <frame_style name="modal_dialog_focused" geometry="modal">
 	<piece position="entire_background" draw_ops="entire_background_focused" />
-	<piece position="titlebar" draw_ops="titlebar_focused" />
+	<piece position="titlebar" draw_ops="titlebar_fill_focused" />
 	<piece position="title" draw_ops="title_focused" />
 	<piece position="overlay" draw_ops="border_focused" />
 	<button function="close" state="normal" draw_ops="close_focused" />
diff --git a/desktop-themes/Menta/metacity-1/metacity-theme-3.xml b/desktop-themes/Menta/metacity-1/metacity-theme-3.xml
index 005e0f21..a91be4ea 100644
--- a/desktop-themes/Menta/metacity-1/metacity-theme-3.xml
+++ b/desktop-themes/Menta/metacity-1/metacity-theme-3.xml
@@ -77,9 +77,9 @@
 </frame_geometry>
 
 <frame_geometry name="border" has_title="false" rounded_top_left="false" rounded_top_right="false" parent="normal" >
-	<distance name="left_width" value="1" />
-	<distance name="right_width" value="1" />
-	<distance name="bottom_height" value="1" />
+	<distance name="left_width" value="6" />
+	<distance name="right_width" value="6" />
+	<distance name="bottom_height" value="6" />
 	<border name="title_border" left="10" right="10" top="0" bottom="0" />
 	<border name="button_border" left="0" right="0" top="0" bottom="0"/>
 	<distance name="title_vertical_pad" value="1" />
@@ -94,8 +94,8 @@
 	<border name="title_border" left="0" right="0" top="0" bottom="0" />
 </frame_geometry>
 
-<frame_geometry name="modal" title_scale="small" hide_buttons="true" rounded_top_left="false" rounded_top_right="false" parent="small">
-	<distance name="title_vertical_pad" value="5"/>
+<frame_geometry name="modal" title_scale="medium" hide_buttons="true" rounded_top_left="false" rounded_top_right="false" parent="small">
+	<distance name="title_vertical_pad" value="10"/>
 </frame_geometry>
 
 <frame_geometry name="attached" title_scale="medium" hide_buttons="true" rounded_top_left="4" rounded_top_right="4" rounded_bottom_left="4" rounded_bottom_right="4" parent="normal">
@@ -645,7 +645,7 @@
 	<button function="unstick" state="pressed"><draw_ops></draw_ops></button>
 </frame_style>
 
-<frame_style name="dialog_focused" geometry="nobuttons">
+<frame_style name="dialog_focused" geometry="normal">
 	<piece position="entire_background" draw_ops="entire_background_focused" />
 	<piece position="titlebar" draw_ops="rounded_titlebar_focused" />
 	<piece position="title" draw_ops="title_focused" />
@@ -677,7 +677,7 @@
 	<button function="unstick" state="pressed"><draw_ops></draw_ops></button>
 </frame_style>
 
-<frame_style name="dialog_unfocused" geometry="nobuttons">
+<frame_style name="dialog_unfocused" geometry="normal_unfocused">
 	<piece position="entire_background" draw_ops="entire_background_unfocused" />
 	<piece position="titlebar" draw_ops="rounded_titlebar_unfocused" />
 	<piece position="title" draw_ops="title_unfocused" />
@@ -710,7 +710,7 @@
 
 <frame_style name="modal_dialog_focused" geometry="modal">
 	<piece position="entire_background" draw_ops="entire_background_focused" />
-	<piece position="titlebar" draw_ops="titlebar_focused" />
+	<piece position="titlebar" draw_ops="titlebar_fill_focused" />
 	<piece position="title" draw_ops="title_focused" />
 	<piece position="overlay" draw_ops="border_focused" />
 	<button function="close" state="normal" draw_ops="close_focused" />
-- 
2.21.0

